src/Orleans/Runtime/GrainReference.cs

```diff
@@ -339,8 +339,8 @@ protected void InvokeOneWayMethod(int methodId, object[] arguments, InvokeMethod
                 return default(T);
             }
 
-            resultTask = OrleansTaskExtentions.ConvertTaskViaTcs(resultTask);
-            return (T) await resultTask;
+            object result = await resultTask.ConfigureAwait(false);
```

Comment:
@jdom , why `ConfigureAwait(false);`?
This code runs both on the client and on the silo, and we don't do `ConfigureAwait(false);` on the silo, ever.

What is it good for? (I mean, I know in general what ConfigureAwait(false);  is good for. Why did you put it here?)